examples/c-comparisons/script.jl

```diff
@@ -0,0 +1,139 @@
+# # Binary Classification with Laplace approximation
+#
+# This example demonstrates how to carry out non-conjugate Gaussian process
+# inference using the Laplace approximation.
+#
+# For a basic introduction to the functionality of this library, please refer
+# to the [User Guide](@ref).
+#
+# ## Setup
+
+using ApproximateGPs
+using LinearAlgebra
+using Distributions
+using LogExpFunctions: logistic, softplus, invsoftplus
+using Zygote
+using Optim
+
+using Plots
+default(; legend=:outertopright, size=(700, 400))
+
+using Random
+Random.seed!(1);
+
+# ## Generate training data
+#
+# We create a binary-labelled toy dataset:
+
+Xgrid = -4:0.1:29  # for visualization
+X = range(0, 23.5; length=48)  # training inputs
+f(x) = 3 * sin(10 + 0.6x) + sin(0.1x) - 1  # latent function
+fs = f.(X)  # latent function values at training inputs
+
+lik = BernoulliLikelihood()  # has logistic invlink by default
+## could use other invlink, e.g. normcdf(f) = cdf(Normal(), f)
+
+invlink = lik.invlink
```

Comment:
```suggestion
invlink = lik.invlink # logistic function
```

It cannot hurt to remind what it is